templates/repo/diff/box.tmpl

```diff
@@ -94,6 +94,10 @@
 							{{if $file.IsProtected}}
 								<span class="ui basic label">{{$.i18n.Tr "repo.diff.protected"}}</span>
 							{{end}}
+							{{if not (or $file.IsIncomplete $file.IsBin $file.IsSubmodule)}}
+								<a class="ui tiny button unescape-diff-button">{{$.i18n.Tr "repo.unescape_control_characters"}}</a>
+								<a class="ui tiny button escape-diff-button" style="display: none;">{{$.i18n.Tr "repo.escape_control_characters"}}</a>
```

Comment:
The logic for `escape-button` and `escape-diff-button` seems very similar.

Can they share the same JS code, and use an unique name, eg: `unicode-escape-content` / `unicode-unescape-content` ?